apache-jsp/pom.xml

```diff
@@ -103,10 +103,9 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-http</artifactId>
+      <groupId>org.eclipse.jetty.tests</groupId>
+      <artifactId>jetty-http-test</artifactId>
```

Comment:
I think `jetty-http-tools` is a better name
Something called `jetty-http-test` should test jetty-http
Something called `jetty-http-tools` contains tools built from jetty-http